File: ios/ReadApp/Services/MangaImageNormalizationRules.swift
import Foundation

struct MangaImageHostRewrite {
    let fromSuffix: String
    let toSuffix: String
}

enum MangaImageNormalizationRules {
    static let hostRewrites: [MangaImageHostRewrite] = [
        MangaImageHostRewrite(fromSuffix: "bzmh.net", toSuffix: "bzcdn.net"),
        MangaImageHostRewrite(fromSuffix: "godamanga.com", toSuffix: "cncover.godamanga.online")
    ]

    static let preferSignedHosts: [String] = [
        "kkmh.com"
    ]
}

File: android/src/main/java/com/readapp/data/manga/MangaImageNormalizationRules.kt
package com.readapp.data.manga

data class MangaImageHostRewrite(
    val fromSuffix: String,
    val toSuffix: String
)

object MangaImageNormalizationRules {
    val hostRewrites = listOf(
        MangaImageHostRewrite(fromSuffix = "bzmh.net", toSuffix = "bzcdn.net")
    )

    val preferSignedHosts = listOf(
        "kkmh.com"
    )
}
